docs/proposed/magic-folder/remote-to-local-sync.rst

```diff
@@ -371,7 +371,7 @@ this procedure for an overwrite in response to a remote change:
    to obtain an initial classification as an overwrite or a
    conflict. (This takes as input the ``last_downloaded_uri``
    field from the directory entry of the changed ``foo``.)
-3. Set the ``mtime`` of the replacement file to be *T* seconds
+3. Set the ``mtime`` of the replacement file to be at least *T* seconds
```

Comment:
On lines 469-470, change "which is not equal to the timestamp *T* seconds before step 4a" to "which is after the timestamp *T* seconds before step 4a".

On lines 569-570, change "which is not equal to the timestamp *T* seconds before \`ReplaceFileW\`_ is called" to "which is after the timestamp *T* seconds before \`ReplaceFileW\`_ is called".